### PR TITLE
Update BF16 amp list

### PR DESCRIPTION
**Pull request by arlesniak**
### PR types

Others
### PR changes

Others
### Describe

Update BF16 amp list

### CODE REVIEW

**Review by jakpiase — DISMISSED** [review]
LGTM

File: python/paddle/fluid/contrib/mixed_precision/bf16/amp_lists.py [hunk 1]
```diff
@@ -83,7 +83,12 @@ def _update_list(self):
 bf16_initializer_list = {'fill_constant', 'uniform_random'}
 
 # always bf16
-bf16_list = {'elementwise_add', 'mul'}
```
Comment (wozna) [inline python/paddle/fluid/contrib/mixed_precision/bf16/amp_lists.py:86]: Is `elementwise_add` BF16 not working properly? Can we add this to this gray list?

Comment (arlesniak) [reply python/paddle/fluid/contrib/mixed_precision/bf16/amp_lists.py:86]: good catch, `elementwise_add` is working OK, it should be on gray list. The gray_list is imported to save space and contains `elementwise_add` already.

**Review by jczaja — APPROVED** [review]
LGTM

**Review by jakpiase — APPROVED** [review]
LGTM

**Review by lidanqing-vv — APPROVED** [review]
LGTM

**Review by Aganlengzi — APPROVED** [review]
LGTM

**Pull request merged**